eos/rare-b-decays/b-to-kstar-gamma-bfs2004_TEST.cc

```diff
@@ -226,23 +229,22 @@ class BToKstarGammaBobethCompatibilityTest :
             observables.push_back(Observable::make("B->K^*gamma::BR_CP_specific;q=d",  p, k, o));
             observables.push_back(Observable::make("B->K^*gamma::S_K^*gamma;q=d", p, k, o));
             observables.push_back(Observable::make("B->K^*gamma::C_K^*gamma;q=d", p, k, o));
-
-            // This line is commented out since it requires a fix, see Github issue #456
-            // observables.push_back(Observable::make("B->K^*gamma::A_I", p, k, o));
+            observables.push_back(Observable::make("B->K^*gamma::A_I", p, k, o));
 
             std::string filename(EOS_SRCDIR "/eos/rare-b-decays/exclusive-b-to-s-gamma_TEST-btokstargamma.data");
 #ifdef EOS_GENERATE_TEST_DATA
             {
                 std::cout << "-- GENERATING test case data for B->K^*gamma --" << std::endl;
-                RandomNumberGenerator rng;
+                gsl_rng * rng = gsl_rng_alloc(gsl_rng_taus);
```

Comment:
``gsl_rng_taus`` is deprecated in favour of ``gsl_rng_taus2`` according to the GSL manual.

Reply:
Good point, I missed that